src/components/AutocompleteDropdown.tsx

```diff
@@ -320,7 +322,10 @@ function AutocompleteDropdown({
   )
 }
 
-export default React.memo(AutocompleteDropdown)
+// export default React.memo(AutocompleteDropdown)
+export default function <T>(): React.ComponentType<Props<T>> {
+  return React.memo<Props<T>>(AutocompleteDropdown)
+}
```

Comment:
I actually only did this to get Flow to be happy. I believe with typescript, this can stay as 

```tsx
export default React.memo(AutocompleteDropdown)
```

Reply:
Can't pass the type if I do this?

Reply:
Ah, after a few mins reading docs, you can do

```tsx
export default React.memo(AutocompleteDropdown) as typeof AutocompleteDropdown
```

and then usage becomes

```tsx
<AutocompleteDropdown<MyDataTypeGoesHere>
  {...props}
/>
```

Reply:
Ah, just the first part thats important looks like